controllers/user.js

```diff
@@ -14,20 +14,10 @@ class Users {
     * @param {object} req - The request object.
     * @param {object} res - The response object.
     */
-  static getUser(req, res) {
-    User.findById(req.params.userId)
-      .then((user) => {
-        if (!user) {
-          return res.status(404).json({
-            message: 'User Not Found'
-          });
-        }
-        return res.status(200).json({
-          message: 'User Found',
-          user
-        });
-      })
-      .catch(error => res.status(400).send(error));
+  static async getUser(req, res) {
+    const user = await User.findOne({ where: { id: req.params.userId } });
+    // eslint-disable-next-line no-unused-expressions
+    !user ? res.status(404).json({ message: `No user with userId ${req.params.userId}, Please make sure you have the correct userId.` }) : res.status(200).json({ message: 'success', user });
```

Comment:
The standard we are following currently is to allow only successful operations be evident in the controller.

Also, we are handling errors in the middlewares as I have previously suggested.

Check the` register` controller for clarification.

Reply:
Ok. I'll implement that now

Reply:
Feedback implemented